target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -172,6 +173,7 @@ case "$FIRMWARE" in
 	netgear,wndap360)
 		caldata_extract "art" 0x5000 0xeb8
 		;;
+	trendnet,tew-673gru|\
```

Comment:
Please maintain alphabetical order.

Reply:
Thanks for catching that (and apologies you needed to).

Reply:
No need to apologize, thanks for addressing it.